hermes/bin/src/runtime_extensions/hermes/http_gateway/mod.rs

```diff
@@ -1,13 +1,20 @@
 //! HTTP Gateway
 
+use std::sync::OnceLock;
+
 use gateway_task::spawn;
 use rusty_ulid::Ulid;
 
+use crate::vfs::Vfs;
+
 mod event;
 mod gateway_task;
 /// Gateway routing logic
 mod routing;
 
+/// Virtual file system
+static VFS: OnceLock<Vfs> = OnceLock::new();
```

Comment:
No, seems it's not correct.
As we dont have a global `Vfs` instance for all apps and each app has its own unique `Vfs` instance.